### PR TITLE
Hardcoded mails

### PR DESCRIPTION
**Pull request by nirae**
Set a "last_chance_mail" line in the config.yml file for send a mail even if gitlab is unavailable

### CODE REVIEW

File: gitlab/gitlab.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/factorysh/minasan/metrics"
 	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
```
Comment (athoune) [inline gitlab/gitlab.go]: No `viper` import in code. It's as evil as global variable.

**Review by athoune — CHANGES_REQUESTED** [review]
Viper is only for `cmd` package.

File: minasan/minasan.go [hunk 2]
```diff
@@ -23,14 +23,14 @@ type Minasan struct {
 }
 
 // Targets return mails, group, project from a mail name
-func (m *Minasan) Targets(mailName string) ([]string, string, string, error) {
+func (m *Minasan) Targets(mailName, lastChanceMail string) ([]string, string, string, error) {
```
Comment (athoune) [inline minasan/minasan.go]: `lastChanceMail` is a setting, not an argument. It's belong to `Minasan` struct.

**Review by athoune — CHANGES_REQUESTED** [review]

**Comment by nirae** [comment]
Think it's good!

**Review by athoune — APPROVED** [review]

**Pull request merged**